packages/pluggableWidgets/timeline-web/src/preview/preview.ts

```diff
@@ -0,0 +1,7 @@
+export const lineAndDotSVG = `<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
🎄  Move this file to `packages/pluggableWidgets/timeline-web/src/assets` to keep it consistent with FAB.

Reply:
👍